game_patch/purefaction/pf.cpp

```diff
@@ -151,12 +167,16 @@ static void process_pf_player_stats_packet(const void* data, size_t len, [[ mayb
 
     const std::byte* player_stats_ptr = static_cast<const std::byte*>(data) + sizeof(stats_packet);
     for (int i = 0; i < stats_packet.player_count; ++i) {
-        pf_player_stats_packet::player_stats in_stats;
+        pf_player_stats_packet::player_stats in_stats{};
         std::memcpy(&in_stats, player_stats_ptr, sizeof(in_stats));
         player_stats_ptr += sizeof(in_stats);
         auto* player = rf::multi_find_player_by_id(in_stats.player_id);
         if (player) {
             auto& stats = *static_cast<PlayerStatsNew*>(player->stats);
+            // Ignore `is_pure`.  Why is it in `player_stats`? 

```

Comment:
I guess so. Can PF fail auth after a player has authed?

Reply:
I think so. It checks it before spawning so I assume it can de-auth you on every respawn, but in practice it's probably only evaluated during level change